tests/transports-integrations/tests/integrations/test_anthropic.py

```
@@ -0,0 +1,589 @@
+        )
+
+        # Should automatically choose to use the calculator
+        assert_has_tool_calls(response, expected_count=1)
+        tool_calls = extract_tool_calls(response)
+        assert tool_calls[0]["name"] == "calculate"
+
+    @skip_if_no_api_key("anthropic")
+    def test_07_image_url(self, anthropic_client, test_config):
+        """Test Case 7: Image analysis from URL"""
+        # Download image and convert to base64 for Anthropic
+        response_img = requests.get(IMAGE_URL)
```

Ruff (0.11.9)

Probable use of `requests` call without timeout

(S113)